xmppserver/src/main/java/org/jivesoftware/openfire/pubsub/PubSubEngine.java

```diff
@@ -48,6 +64,12 @@
 
     private static final Logger Log = LoggerFactory.getLogger(PubSubEngine.class);
 
+	public static final SystemProperty<Boolean> PUBSUB_NONREGUSER = SystemProperty.Builder.ofType(Boolean.class)
+		    .setKey("xmpp.pubsub.createnode.nonregistereduser")
```

Comment:
in `openfire_i18n.properties`, add a line like this:

`system_property.xmpp.pubsub.createnode.nonregistereduser=Allows PubSub nodes to be created for JIDs that do not represent an existing user.`

Reply:
updated the i18.properties file with description